float64 distance
string status_flag
